pylib/cqlshlib/cql3handling.py

```diff
@@ -1611,32 +1611,23 @@ def permission_completer(ctxt, _):
 
 @completer_for('username', 'name')
 def username_name_completer(ctxt, cass):
-    def maybe_quote(name):
-        if CqlRuleSet.is_valid_cql3_name(name):
-            return name
-        return "'%s'" % name
-
     # disable completion for CREATE USER.
     if ctxt.matched[0][1].upper() == 'CREATE':
         return [Hint('<username>')]
 
     session = cass.session
-    return [maybe_quote(list(row.values())[0].replace("'", "''")) for row in session.execute("LIST USERS")]
+    return map(maybe_escape_name, [row['name'] for row in session.execute("LIST USERS")])
```

Comment:
What about using the wrapper `protect_name`?
https://github.com/datastax/python-driver/blob/master/cassandra/metadata.py#L1567

Reply:
`protect_name` internally calls `maybe_escape_name`. Maybe I'm missing something, but what would be the benefit of using `protect_name`?

Reply:
Basically, just use the "higher abstraction" provided in the driver? It is just a nit, but I am curious to why `protect_name` even exists.

Reply:
to me it feels like it's more of a legacy method name : https://github.com/datastax/python-driver/commit/0959b5dc1662d6957f2951738d0a4f053ac78f66 and not really a higher abstraction. But I don't have a lot of context on the history of these method names